### ISSUE TITLE
Security options with `:` as a separator are deprecated

### ISSUE THREAD

**Issue by hesco** [docker/docker#23186]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Output of `docker version`:**

```
# docker version 
Client:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:11:07 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.11.1
 API version:  1.23
 Go version:   go1.5.4
 Git commit:   5604cbe
 Built:        Tue Apr 26 23:11:07 2016
 OS/Arch:      linux/amd64

```

**Output of `docker info`:**

```
# docker info 
Containers: 6
 Running: 3
 Paused: 0
 Stopped: 3
Images: 35
Server Version: 1.11.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 73
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins: 
 Volume: local
 Network: weavemesh bridge null host
Kernel Version: 3.16.0-4-amd64
Operating System: Debian GNU/Linux 8 (jessie)
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 94.6 GiB
Name: dessalines014
ID: APL2:PO4G:DXQG:5DYJ:COK2:3HQG:GYGR:W6AN:FK4B:GT3J:VP2B:M272
Docker Root Dir: /var/lib/docker
Debug mode (client): false
Debug mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No memory limit support
WARNING: No swap limit support
WARNING: No kernel memory limit support
WARNING: No oom kill disable support
WARNING: No cpu cfs quota support
WARNING: No cpu cfs period support

```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

bare metal docker host 

**Steps to reproduce the issue:**
start or stop the docker daemon, 
run most any docker query
or even a weave query

**Describe the results you received:**
WARN[0325] Security options with : as a separator are deprecated and will be completely unsupported in 1.13, use = instead.

**Describe the results you expected:**
a quiet, no warning response to my commands.  

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by bboreham** [comment]
Looks like #22413

**Comment by thaJeztah** [comment]
Yes, this is a duplicate of https://github.com/weaveworks/weave/issues/2328, and fixed on master; to be part of the 1.12 release.
